### PR TITLE
Fix xdr isloate / unisolate playbooks

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-19696

## Description
Fixed the following issues:
- Created new versions for the playbooks from the 6.2 server version.
- Changed deprecated commands for new ones ( which contain inside the polling logic).
- Fixed logic issues in the flow + fixed some conditional tasks.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/4_9_12.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Cortex XDR - Unisolate Endpoint
+- Fixed an issue with the conditional task "Is the endpoint ready for isolation?".
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_9_12.md:4]: which issue? :)

File: Packs/CortexXDR/ReleaseNotes/4_9_12.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Changed deprecated command to the new one "xdr-endpoint-isolate".
+##### Cortex XDR - Isolate Endpoint
+- Added new inputs support.
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_9_12.md:7]: Specify the inputs

File: Packs/CortexXDR/ReleaseNotes/4_9_12.md (hunk 1 above)
Comment (ssokolovich) [reply Packs/CortexXDR/ReleaseNotes/4_9_12.md]: Fixed

File: Packs/CortexXDR/ReleaseNotes/4_9_12.md (hunk 2 above)
Comment (ssokolovich) [reply Packs/CortexXDR/ReleaseNotes/4_9_12.md]: Fixed

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Isolate_Endpoint6_2.yml [hunk 3]
```diff
@@ -21,6 +21,7 @@ tasks:
       '#none#':
       - "2"
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_-_Isolate_Endpoint6_2.yml:22]: Change the file names to be with the minimum supported version ( add 6.2 for the new ones).

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Isolate_Endpoint6_2.yml [hunk 4]
```diff
@@ -0,0 +1,444 @@
+id: Cortex XDR - Isolate Endpoint
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Isolate_Endpoint6_2.yml:1]: In the file name add "_" before the 6

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Isolate_Endpoint6_2.yml [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+      - "3"
+    scriptarguments:
+      endpoint_id:
+        complex:
+          root: inputs.endpoint_id
+          filters:
+          - - operator: isNotEmpty
+              left:
+                value:
+                  simple: inputs.endpoint_id
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Isolate_Endpoint6_2.yml:63]: Lets changed it as we discussed

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Unisolate_Endpoint6_2.yml [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      endpoint_id:
+        complex:
+          root: inputs.Endpoint_ID
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Unisolate_Endpoint6_2.yml:176]: let's take the input from the get endpoint status task (like what we talked about with the isolate playbook)

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**